swim/memberlist.go

```diff
@@ -324,3 +324,17 @@ func (m *memberlist) String() string {
 func (m *memberlist) Iter() *memberlistIter {
 	return newMemberlistIter(m)
 }
+
+func (m *memberlist) GetActiveMemberAddresses() []string {
+	var active []string
+
+	m.members.RLock()
```

Comment:
True. But looking at the code that is already in this package I see this same pattern over and over again. So I would vote for consistency at this point and in an other story figure out what the performance difference is between this method of locking vs only locking a copy operation.

1. [`NumPingableMembers`](https://github.com/uber/ringpop-go/blob/feature/get-members/swim/memberlist.go#L134-L144)
2. [`RandomPingableMembers`](NumPingableMembers)
3. [`GetMembers`](https://github.com/uber/ringpop-go/blob/feature/get-members/swim/memberlist.go#L168-L176) (not the method being added in this PR)